docs/src/docs/asciidoc/getting-started.adoc

```diff
@@ -1,21 +1,57 @@
 [[getting-started]]
 = Getting Started
-:toc: left
-:toclevels: 1
 
-This page is under construction.
+If you are just getting started with Spring Authorization Server, the following sections will walk you through creating your first application.
 
 [[system-requirements]]
 == System Requirements
 
-This section is under construction.
+Spring Authorization Server requires a Java 8 or higher Runtime Environment.
 
 [[installing-spring-authorization-server]]
 == Installing Spring Authorization Server
 
-This section is under construction.
+Spring Authorization Server can be used anywhere you're already using https://docs.spring.io/spring-security/reference/prerequisites.html[Spring Security].
+
+The easiest way to begin using Spring Authorization Server is by creating a https://spring.io/projects/spring-boot[Spring Boot]-based application.
+You can use https://start.spring.io[start.spring.io] to generate a basic project or use the https://github.com/spring-projects/spring-authorization-server/tree/main/samples/default-authorizationserver[default authorization server sample] as a guide.
+Then simply add Spring Authorization Server as a dependency, as in the following example:
+
+[[maven-dependency]]
+.Maven
+[source,xml,role="primary",subs="attributes,verbatim"]
+----
+<dependency>
+    <groupId>org.springframework.security</groupId>
+    <artifactId>spring-security-oauth2-authorization-server</artifactId>
+    <version>{spring-authorization-server-version}</version>
+</dependency>
+----
+
+[[gradle-dependency]]
+.Gradle
+[source,gradle,role="secondary",subs="attributes,verbatim"]
+----
+implementation "org.springframework.security:spring-security-oauth2-authorization-server:{spring-authorization-server-version}"
+----
+
+TIP: See https://docs.spring.io/spring-boot/docs/current/reference/html/getting-started.html#getting-started.installing[Installing Spring Boot] for more information on using Spring Boot with Maven or Gradle.
 
 [[developing-your-first-application]]
-== Developing Your First Spring Authorization Server Application
+== Developing Your First Application
+
+To get started, you will need several components defined as an `@Bean` in a Spring `@Configuration`. These components can be defined as in the example configuration below:
```

Comment:
Instead of "several components" update to something like "minimum required components"